### ISSUE TITLE
How to finetune/improve model

### ISSUE THREAD

**Issue by kunibald413** [pkhungurn/talking-head-anime-3-demo#28]
What would be the steps to take to improve the model? 

Specifically is there a way to finetune the face morpher to be able to handle more faces with greater accuracy? 

Is there a writeup or some training code available?
